optas/models.py

```diff
@@ -200,6 +230,22 @@ def link_names(self):
     def actuated_joint_names(self):
         return [jnt.name for jnt in self._urdf.joints if jnt.type != 'fixed']
 
+    @property
+    def param_joint_names(self):
```

Comment:
Could you use the full name here, i.e. `parameter_joint_names`. For variables it is fine to abbreviate but for class properties and methods I prefer to keep them as descriptive as possible.

Reply:
done!